docs/_documentations/mdt-intellij-getting-started.md

```diff
@@ -0,0 +1,69 @@
+2. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
+   - On macOS, go to **Main**>**Preferences**.
+   - On Windows, go to **File**>**Settings**.
+3. Click the **Manage Repositories, Configure Proxy or Install Plugin from Disk** gear icon to open **Plugins**. Click the **Manage plugin repositories** icon. From the menu, click **Install Plugin from Disk...**. A window appears that displays the files on your computer.
+4. Choose the `codewind-intellij-0.9.0.zip` file and click **Open**.
+
+## Creating your first project from a template
+1. Start IntelliJ. From the welcome page, click **Create New Project**. The **New Project** window opens.
+2. Create a Codewind project by clicking **Codewind**.
+   - If Codewind isn't installed yet, click **Install Codewind** to complete the installation.
+   - If Codewind isn't running yet, click **Start Codewind**. A **Starting Codewind** window appears.
+3. After Codewind is installed and started, click **Next**. A list of available templates appears.
+   - Currently, IntelliJ supports only the Java language templates.
+4. Click the template of your choice. Then, click **Next**.
+5. Enter the **Project name:** field to name your project and enter the **Project location:** field to choose where to put it.
+6. Click **Finish**. A window for your project appears.
+7. IntelliJ detects a `pom.xml` file and asks if you want to add the file as a Maven project. In the **Non-managed pom.xml file found:** window, click **Add as Maven Project** to access your project source files and create the correct project structure.
+   - The **Non-managed pom.xml file found:** window might disappear after some time. To open it again, go to the IntelliJ project view and click the `pom.xml` file. From the context menu, click **Add as Maven Project** to create the correct project structure.
+8. To access the Codewind view, click the **Codewind** tab. From the Codewind view, right-click your project to open an application, start a build, and more.
+
+## Creating more projects from templates
+1. From an IntelliJ project that's already open, go to **File**>**New**>**Project...**. The **New Project** window appears.
+2. Click **Next**.
+3. Click the template of your choice and click **Next**. A list of available templates appears.
+   - Currently, IntelliJ supports only the Java language templates.
+4. Enter the **Project name:** and **Project location:** and click **Finish**. The **Open Project** window appears.
+5. To create a new project, click **New Window**. A new window appears for the new project.
+6. IntelliJ detects a `pom.xml` file and asks if you want to add the file as a Maven project. In the **Non-managed pom.xml file found:** window, click **Add as Maven Project** to access your project source files and create the correct project structure.
```

Comment:
Same changes needs to be made here as in the comments for creating your first project.

It might be worth mentioning that unlike in the other IDEs, there is no 'Create New Project" menu item in the codewind view, you need to do it from File > New Project.